api/registry/registry_integration.go

```diff
@@ -81,6 +108,13 @@ func (d *idm) Mount(
 	volumeID, volumeName string,
 	opts *types.VolumeMountOpts) (string, *types.Volume, error) {
 
+	fields := log.Fields{
+		"driverName": d.Name(),
```

Comment:
There should be no reason to log the driver name. It should be set in the context.